signal_scanner_bot/filters.py

```diff
@@ -72,13 +72,7 @@ def _f_retweet_text(status: Status) -> bool:
     return status.text.startswith("RT @")
 
 
-def _f_not_trusted_tweeter(status: Status) -> bool:
-    # Status wasn't sent by a trusted tweeter
-    return status.author.screen_name not in env.TRUSTED_TWEETERS
-
-
 TWITTER_FILTERS: List[Callable[[Status], bool]] = [
-    _f_retweeted,
-    _f_retweet_text,
     _f_not_trusted_tweeter,
+    _f_retweet_text,
```

Comment:
Really smart move to swap these, as the former will filter out way more than the latter on first pass!

Reply:
Oh, yes... definitely 100% meant to do that and thought it through completely, seeing the gains. This was in no way an unintentional change.

Reply:
:joy: :joy: :joy: hey, take the credit either way!